### ISSUE TITLE
Edits are not displayed before chunk reload

### ISSUE THREAD

**Issue by lusu007** [IntellectualSites/FastAsyncWorldEdit#847]
**[REQUIRED] FastAsyncWorldEdit Configuration Files** 

**/fawe debugpaste**: https://athion.net/ISPaster/paste/view/e64973a2c55249cd9e02d6e8c38ee45d

**Required Information**
- FAWE Version Number (`/version FastAsyncWorldEdit`): Bukkit-Official(1.16-540;b4d7562)
- Spigot/Paper Version Number (`/version`): git-Paper-403 (MC: 1.16.4)
- Minecraft Version: 1.16.4  

**Describe the bug**
All actions performed with FAWE work without problems, but they are not displayed. The changes are only visible when the chunks are unloaded and loaded again. Also F3+A is not sufficient.

Can send-before-history be to blamed for this?

**To Reproduce**
1. Execute any action with FAWE (e.g. //replace diamond_block)
2. Changes are not visible
3. Fly away until the chunks are unloaded
4. Fly back to the position where you did the change
5. Changes are now loaded.

**Plugins being used on the server**

Arceon, Builders-Utilities, BungeeTabListPlus, Essentials, FastAsyncWorldEdit (WorldEdit), goBrush, goPaint, HeadDatabase, Hyperverse, LegendEssentials, LegendSchematicScatter, LuckPerms, Vault, VoidGenerator, VoxelSniper, WorldGuard

**Checklist**:

- [x] I included all information required in the sections above
- [x] I made sure there are no duplicates of this report [(Use Search)](https://github.com/IntellectualSites/FastAsyncWorldEdit/issues?q=is%3Aissue)
- [x] I made sure I am using an up-to-date version of [FastAsyncWorldEdit for 1.16.4](https://ci.athion.net/job/FastAsyncWorldEdit-1.16/)
- [x] I made sure the bug/error is not caused by any other plugin

**Comment by aurorasmiles** [comment]
I'm not able to replicate your issue

**Comment by lusu007** [comment]
I made a short clip to illustrate the problem. Its quality is a bit bad because Githubs file limit is at 10MB.

https://user-images.githubusercontent.com/10341994/104454443-26decd80-55a6-11eb-84ca-163b2db538ae.mp4

**Comment by LaurenceBarnes** [comment]
could you try to do //set stone multiple times? I wonder if it is shown then after the 2nd or 3rd time

**Comment by lusu007** [comment]
Executing ``//set stone`` multiple times doesnt help.

**Comment by LaurenceBarnes** [comment]
> I'm not able to replicate your issue

well I can though, on build-560 too.

I noticed that it happens when you do a small edit in a recently edited region. That is how I noticed it. I tested through every dimension we have.

Since I am copying over a lot of progress, whole dozens of chunks are changed. To adjust some things, I used FAWE in those newly pasted chunks to adjust landscape, and then the bug described in that issue occured to me, multiple times at different spots. In regions that haven't been edited in a while (at least a couple hours), it doesn't occur. It is really hard to reproduce.
